pkgs/applications/kde/falkon.nix

```diff
@@ -27,21 +19,14 @@ mkDerivation rec {
   buildInputs = [
     libpthreadstubs libxcb libXdmcp
     qtsvg qttools qtwebengine qtx11extras
-    kwallet
+    kwallet kpurpose karchive kio pyside2 shiboken2
   ] ++ lib.optionals stdenv.isLinux [ qtwayland ];
 
-  nativeBuildInputs = [
-    cmake
-    extra-cmake-modules
-    pkg-config
-    qmake
-    qttools
-    wrapQtAppsHook
-  ];
+  nativeBuildInputs = [ cmake extra-cmake-modules pkg-config qttools wrapQtAppsHook ];
```

Comment:
Nitpick: I think you should keep it as separate lines since changing the format makes it harder to see what you changed. Keeping it as separate lines or keeping it as a single line (if it was already like that) makes the diff easy to read on github.

Reply:
Done. Fixed-up and rebased in order to avoid polluting blame for these files.